src/audio/module_adapter/module/volume/volume.c

```diff
@@ -335,6 +336,7 @@ static void volume_ramp(struct processing_module *mod)
 				}
 			}
 		}
+		cd->copy_needed = true;
```

Comment:
put it outside of the for loop seems faster?

Reply:
if outside loop, then it can't cover continue case, for continue cases, if all channel is continue, then no need update gain.